### PR TITLE
Add support for NOT_EQUAL Compare operator

### PR DESCRIPTION
**Pull request by jd**
Add support for NOT_EQUAL Compare operator
Update to latest proto version

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -26,17 +26,17 @@ commands = py.test --cov=etcd3 tests/
...
-    sed -i -e '/gogoproto/d' etcd3/proto/rpc.proto
-    sed -i -e 's/etcd\/mvcc\/mvccpb\/kv.proto/kv.proto/g' etcd3/proto/rpc.proto
-    sed -i -e 's/etcd\/auth\/authpb\/auth.proto/auth.proto/g' etcd3/proto/rpc.proto
-    sed -i -e '/google\/api\/annotations.proto/d' etcd3/proto/rpc.proto
-    sed -i -e '/option (google.api.http)/,+3d' etcd3/proto/rpc.proto
+    gsed -i -e '/gogoproto/d' etcd3/proto/rpc.proto
+    gsed -i -e 's/etcd\/mvcc\/mvccpb\/kv.proto/kv.proto/g' etcd3/proto/rpc.proto
+    gsed -i -e 's/etcd\/auth\/authpb\/auth.proto/auth.proto/g' etcd3/proto/rpc.proto
+    gsed -i -e '/google\/api\/annotations.proto/d' etcd3/proto/rpc.proto
+    gsed -i -e '/option (google.api.http)/,+3d' etcd3/proto/rpc.proto
```
Comment (kragniz) [inline tox.ini]: Could we remove the gnuisms here instead? Otherwise people have to add an alias on linux machines.

Comment (jd) [inline tox.ini]: Sorry I did not mean to commit. It was just needed locally!

**Comment by jd** [comment]
Pull-request updated, HEAD is now 165479127555af1f9ffd94d906842bf9276656ac

**Comment by kragniz** [comment]
lgtm

**Pull request merged**